ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl/ChainSel.hs

```diff
@@ -1325,8 +1313,6 @@ validateCandidate ::
 validateCandidate chainSelEnv rr chainDiff =
   ledgerValidateCandidate chainSelEnv rr chainDiff >>= \case
     validatedChainDiff
-      | ValidatedDiff.rollbackExceedsSuffix validatedChainDiff ->
```

Comment:
Is this function still used somewhere else?

Reply:
Yes:
https://github.com/IntersectMBO/ouroboros-consensus/blob/9e06d112a0ba39b41093a88ad686d3484d235bfc/ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl/ChainSel.hs#L818-L821
But this will be removed in #1594 (fb6ce1fd87b4c37107290048e53f96015ca6810d).